modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -399,8 +399,16 @@ definition {
 				objectName = "CustomObject170529",
 				pluralLabelName = "Custom Objects 170529");
 
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Name 1",
+				accountEntryType = "Business");
+
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Account Name 2",
+				accountEntryType = "Business");
+
```

Comment:
I'm confused, why are we moving this here? Adding the account entries later shouldn't affect the test.

Reply:
Hey @timpak, I move this step to the beginning of the test for since the workflow needs that the account need to created first. Have you another suggestion for this step?